nautobot/ipam/models.py

```diff
@@ -1103,10 +1095,6 @@ def clean(self):
         if self.type == choices.IPAddressTypeChoices.TYPE_SLAAC and self.ip_version != 6:
             raise ValidationError({"type": "Only IPv6 addresses can be assigned SLAAC type"})
 
-        # If neither `parent` or `namespace` was provided; raise this exception
```

Comment:
I think `self._get_closest_parent` is performing this function adequately

Reply:
`_get_closest_parent` doesn't check for the `self._namespace = None` case currently as I read it, so there may need to be some change made there.

Reply:
Currently `self._namespace` defaults to `get_default_namespace()` when `self._provided_namespace = None` and `self.parent` isn't set.